css/properties/transform-box.json

```diff
@@ -0,0 +1,95 @@
+{
+  "css": {
+    "properties": {
+      "transform-box": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/transform-box",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "55"
+              },
+              {
+                "version_added": "43",
+                "flag": {
+                  "type": "preference",
+                  "name": "svg.transform-box.enabled",
+                  "value_to_set": "true"
+                }
+              },
+              {
+                "version_added": "41",
+                "version_removed": true,
```

Comment:
The removal happened in "43". See https://bugzilla.mozilla.org/show_bug.cgi?id=1175492 .